sdk/docs/iot/README.md

```diff
@@ -93,47 +72,58 @@ Feature | Azure SDK for Embedded C | Description
 
 ## Examples
 
+These examples introduce you to the API calls for a few key features of the Embedded C SDK.
+
+- [IoT Hub Client Initialization](#iot-hub-client-initialization)
+- [Properties](#properties)
+- [Telemetry](#telemetry)
+
+Additional pseudo code examples can be found [here](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/docs/iot/pseudo_code_examples.md). The pseudo code examples are MQTT stack agnostic and can give you a general overview of the API calls and structure needed to use the Azure IoT Embedded C SDK features.
```

Comment:
They might be MQTT stack agnostic but I wouldn't consider them pseudo code. Maybe coding pattern template?
```suggestion
Additional coding pattern examples can be found [here](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/docs/iot/pseudo_code_examples.md). The pseudo code examples are MQTT stack agnostic and can give you a general overview of the API calls and structure needed to use the Azure IoT Embedded C SDK features.
```

Reply:
@jspaith or @danewalton any thoughts on this?